arq/connections.py

```diff
@@ -47,6 +48,10 @@ class RedisSettings:
     sentinel: bool = False
     sentinel_master: str = 'mymaster'
 
+    retry_on_timeout: bool = False
+    retry_on_error: Optional[List[Any]] = None
```

Comment:
Wouldn't this be `Optional[List[Exception]]`?

Reply:
Yup, fixed.  thanks for catching.